NEWS.md

```diff
@@ -1,3 +1,10 @@
+# shinyMobile 1.1.0
```

Comment:
@hypebright: it might be 2.0.0 or even shinyMobile2

Reply:
@DivadNojnarg agreed, this was more like a placeholder. Versions in DESCRIPTION etc also need to be bumped up once that's decided. Let's see how many breaking changes we gather in here to see if a shinyMobile2 is needed 👀